editors/sc-ide/widgets/help_browser.cpp

```diff
@@ -91,10 +81,15 @@ HelpBrowser::HelpBrowser(QWidget* parent): QWidget(parent) {
 
     connect(mWebView, SIGNAL(loadStarted()), mLoadProgressIndicator, SLOT(start()));
     connect(mWebView, SIGNAL(loadFinished(bool)), mLoadProgressIndicator, SLOT(stop()));
+    connect(mWebView, SIGNAL(loadFinished(bool)), this, SLOT(onPageLoad()));
```

Comment:
onPageLoad() can call mLoadProgressIndicator->stop(), we don't need both connections